src/main/java/com/sap/sgs/phosphor/fosstars/model/score/AverageCompositeScore.java

```diff
@@ -87,11 +87,11 @@ public ScoreValue calculate(Value... values) {
     int numberOfSubScores = 0;
     for (Score subScore : subScores) {
       ScoreValue subScoreValue = calculateIfNecessary(subScore, valueSet);
+      scoreValue.usedValues(subScoreValue);
```

Comment:
if the subscoreValue is not applicable, should it be considered as used?

Reply:
Yes, please see #174. A score should contain info about all considered sub-scores even if they are N/A.